### DISCUSSION TITLE
CapacitorModel

### DISCUSSION THREAD

**Discussion by Angelo50** [SpiceSharp/SpiceSharp#179] [General]
Hi Sven,

I finally removed my previous question since after some research, it was irrelevent

I'm trying to implement a polirized transistor like that:

C2 N005 N004 2.2µ V=50 Irms=11m Rser=2.26084 Lser=1.38496n mfg="Würth Elektronik" pn="865230640006 WCAP-AS5H 4x5,5" type="Al electrolytic"

but all the SetParameter in spicesharp are rejected
var CapModel = new CapacitorModel("CapSimulation");
        CapModel.SetParameter("v", 16);
        CapModel.SetParameter("irms", 0.011);
        CapModel.SetParameter("rser", 2.26084);
        CapModel.SetParameter("lser", 0.00000000138496);
        CapModel.SetParameter("mfg", "Electrolitik");
        CapModel.SetParameter("type", "Al electrolytic");

if you could help me on that

Thanks again for you time

Angelo.

**Comment by svenboulanger** [comment]
It looks like you are using a model that is meant for LTspice? Most simulators have branched off Spice 3f5 with different/extra parameters. 

- `V` and `irms` are the ratings of the transistor. I don't think they are used in the model equations.
- `rser` and `lser` are what I assume parasitics that can be modeled using resistors and inductors. They are most likely for convenience.
- `mfg` and `type` are transistor metadata that have no effect on the model equations.

Yours sincerely,
Sven

**Comment by Angelo50** [comment]
Thanks a lot for your answer

Angelo.
